React/finger-exercises/react/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,22 @@
+import isArray from './utils';
+
+export function min(...data) {
+  if (data.length === 0) { 
+    return undefined;
```

Comment:
remember what I told you about returning `undefined` so explicitly. if you don't explicitly return something, any javascript function will return `undefined` regardless. Try to think of a way to re write what you already have (which is good) but removing the `if` statement that returns `undefined`.

Reply:
👍